src/FormBuilder/resources/js/form-builder/src/components/onboarding/transfer/TransferDialog.tsx

```diff
@@ -29,30 +29,29 @@ function Confirmation({handleTransferConfirmation}) {
 
     function handleInputChange(e) {
         e.persist();
-        setState(prev => ({...prev, input: e.target.value}))
+        setState((prev) => ({...prev, input: e.target.value}));
     }
 
     function handleCheckboxChange(e, name) {
         e.persist();
-        setState(prev => ({...prev, [name]: e.target.checked}))
+        setState((prev) => ({...prev, [name]: e.target.checked}));
     }
 
     return (
         <>
-            <div className="givewp-dialog-title">
-                {__('Transferring donation data to the upgraded form', 'give')}
-            </div>
+            <div className="givewp-dialog-title">{__('Transferring donation data to the upgraded form', 'give')}</div>
 
             <div className="givewp-dialog-content">
                 <Interweave
-                    content={__('Type <strong>transfer</strong> to confirm transfer of donation data for the form selected. This means all blocks, shortcodes, and the form URL will automatically redirect to the upgraded form.', 'give')} />
+                    content={__(
```

Comment:
This is just a info, you don't have to change anything.
We should probably move away from `Interweave` and use an already baked-in solution for this kind of stuff - https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/#createinterpolateelement

Reply:
@alaca I didn't change it intentionally. It was my IDE auto-formatting because it seems this file wasn't using our code standards previously.